how-to/run-kata-with-k8s.md

```diff
@@ -1,64 +1,86 @@
 
-Once you have installed CRI-O, you need to modify the CRI-O configuration
-with information about different container runtimes. By default, we choose
-`runc`, but in this case we also specify Kata Containers runtime to run
-__untrusted__ workloads. In other words, this defines an alternative runtime
-to be used when the workload cannot be trusted and a higher level of security
-is required. An additional flag can be used to let CRI-O know if a workload
-should be considered _trusted_ or _untrusted_ by default.
-For further details, see the documentation
-[here](https://github.com/kata-containers/documentation/blob/master/design/architecture.md#mixing-vm-based-and-namespace-based-runtimes).
+An equivalent shim implementation for CRI-O is planned.
 
-Additionally, we need CRI-O to perform the network namespace management.
-Otherwise, when the VM starts the network will not be available.
+### CRI-O
+For CRI-O installation instructions, refer to the [CRI-O Tutorial](https://github.com/kubernetes-incubator/cri-o/blob/master/tutorial.md) page.
 
-The following is an example of how to modify the `/etc/crio/crio.conf` file
-in order to apply the previous explanations, and therefore get Kata Containers
-runtime to invoke by CRI-O.
+The following sections show how to set up the CRI-O configuration file (default path: `/etc/crio/crio.conf`) for Kata.
 
+Unless otherwise stated, all the following settings are specific to the `crio.runtime` table:
 ```toml
 # The "crio.runtime" table contains settings pertaining to the OCI
 # runtime used and options for how to set up and manage the OCI runtime.
 [crio.runtime]
-manage_network_ns_lifecycle = true
+```
+A comprehensive documentation of the configuration file can be found [here](https://github.com/cri-o/cri-o/blob/master/docs/crio.conf.5.md).
+
+> **Note**: After any change to this file, the CRI-O daemon have to be restarted with:
+>````
+>$ sudo systemctl restart crio
+>````
+
+#### Kubernetes Runtime Class (CRI-O v1.12+)
+The [Kubernetes Runtime Class](https://kubernetes.io/docs/concepts/containers/runtime-class/)
+is the preferred way of specifying the container runtime configuration to run a Pod's containers.
+To use this feature, Kata must added as a runtime handler with:
+
+```toml
+[crio.runtime.runtimes.kata-runtime]
+  runtime_path = "/usr/bin/kata-runtime"
+  runtime_type = "oci"
+```
 
+You can also add multiple entries to specify alternatives hypervisors, e.g.:
+```toml
+[crio.runtime.runtimes.kata-qemu]
+  runtime_path = "/usr/bin/kata-runtime"
+  runtime_type = "oci"
+
+[crio.runtime.runtimes.kata-fc]
+  runtime_path = "/usr/bin/kata-runtime"
+  runtime_type = "oci"
+```
+
+#### Untrusted annotation (until CRI-O v1.12)
```

Comment:
I suspect we can and should drop the whole 'untrusted annotation' stuff from all of our docs (and, code if there is any). It has been replaced by the `RuntimeClass`...

Reply:
Thank you @grahamwhaley . 
There's also the  `## Run a Kubernetes pod with Kata Containers`  section 
that still needs to be updated with RuntimeClass, but I prefer to postpone that in a next PR and re-use the [Testing Kata Containers](https://github.com/kata-containers/documentation/blob/master/install/minikube-installation-guide.md#testing-kata-containers) in the minikube install guide.